.github/workflows/snyk.yml

```
@@ -0,0 +1,82 @@
+name: security-snyk
+
+on:
+  push:
+    branches: [ main, master, '**/*' ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)